kibana/templates/deployment.yaml

```diff
@@ -121,3 +129,29 @@ spec:
             mountPath: /usr/share/kibana/config/{{ $path }}
             subPath: {{ $path }}
           {{- end -}}
+{{- if .Values.dashboardImport.enabled }}
+      initContainers:
+      - name: {{ .Chart.Name }}-dashboardimport
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        securityContext:
+{{ toYaml .Values.securityContext | indent 10 }}
+        command: ["/bin/bash"]
+        args:
+          - "-c"
+          - "/tmp/dashboardImport.sh"
+        env:
```

Comment:
For this container to properly start up it would also need to include the full environment variables and kibana.yml configuration used for the main container along with any secretMounts. Basically it would need to look identical to the main Kibana container + the extra dashboard and import scripts